inertia/http.py

```diff
@@ -234,8 +240,8 @@ def clear_history(request):
     request.session[INERTIA_SESSION_CLEAR_HISTORY] = True
 
 
-def inertia(component):
-    def decorator(func):
+def inertia(component: str):
+    def decorator(func: Callable[..., HttpResponse | InertiaResponse | dict[str, Any]]):
```

Comment:
For decorators it, would be ideal to use `ParamSpec` and TypeVar to ensure the types are correct in the project where it is used. But I suppose this could be added in future.

Reply:
Didn't know about `ParamSpec`, I'll see if I can use that instead, it does seem useful in this case. Thanks again for the review!

Reply:
It's very handy for decorators as it allows you to use the args and kwargs without knowing what they are. I can always add them in a future PR too once the setup for typing has been added too. My pleasure thanks for you work.